pkg/serviceaccount/jwt.go

```diff
@@ -133,6 +135,8 @@ func (j *jwtTokenAuthenticator) AuthenticateToken(token string) (user.Info, bool
 			return key, nil
 		})
 
+		claims, _ := parsedToken.Claims.(jwt.MapClaims)
```

Comment:
actually, I opened https://github.com/kubernetes/kubernetes/pull/28542 with a test for the invalid token case